tinygrad/lazy.py

```diff
@@ -87,32 +87,29 @@ def create_lazybuffer(device:str, shape:Union[ShapeTracker, Tuple[int, ...]], op
 
 class LazyBuffer:
   __deletable__ = ('op',)
-  def __init__(self, device:str, st:ShapeTracker, optype:OpType, op:LazyOp, dtype:DType):
+  def __init__(self, device:str, st:ShapeTracker, optype:OpType, src:Union[LazyOp, RawBuffer], dtype:DType):
```

Comment:
I'd make a different constructor for this.

Reply:
WDYT of a `fromCPU` method that takes the `np.ndarray`? The current `RawBuffer` does not remember what was its device